### PR TITLE
regroup kindergeld

### PR DESCRIPTION
**Pull request by Eric-Sommer**
### What problem do you want to solve?

Regroup Kindergeld params according to #148 and adjusted code accordingly.

### CODE REVIEW

File: gettsim/taxes/kindergeld.py [hunk 1]
```diff
@@ -13,21 +13,16 @@ def kindergeld(tax_unit, params):
...
     ).cumsum()
 
-    kg_amounts = {
-        1: params["kgeld1"],
-        2: params["kgeld2"],
-        3: params["kgeld3"],
-        4: params["kgeld4"],
-    }
-    tax_unit["kindergeld_m_basis"] = tax_unit["kindeld_anspruch"].replace(kg_amounts)
+    tax_unit["kindergeld_m_basis"] = tax_unit["kindeld_anspruch"].replace(
```
Comment (MaxBlesch) [inline gettsim/taxes/kindergeld.py:20]: Maybe here a little comment on how this works w.r.t. the order of the children?

File: gettsim/benefits/unterhalt.py [hunk 2]
```diff
@@ -32,13 +32,13 @@ def uhv_since_07_2017(tax_unit, params, kindergeld_params):
     # Amounts depend on age
     tax_unit.loc[
         (tax_unit["alter"] < 6) & tax_unit["alleinerziehend"], "unterhaltsvors_m"
-    ] = (params["uhv6_amount"] - kindergeld_params["kgeld1"])
+    ] = (params["uhv6_amount"] - kindergeld_params["kindergeld"][1])
```
Comment (MaxBlesch) [inline gettsim/benefits/unterhalt.py:35]: We should write, why it is the amount for the first kid. I don't see it myself!

Comment (Eric-Sommer) [reply gettsim/benefits/unterhalt.py:35]: that's the ruling...is already commented in line 29f.

Comment (MaxBlesch) [reply gettsim/benefits/unterhalt.py:35]: Okay, sorry!

**Pull request merged**